addons/l10n_pe_website_sale/views/templates.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="partner_info" name="Peruvian partner">
+
+        <!-- show identification type -->
+        <div t-attf-class="mb-3 #{error.get('l10n_latam_identification_type_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+            <t t-if="partner.can_edit_vat()">
```

Comment:
AFAIK, we already have a `can_edit_vat` rendering value holding this value when rendering the `website_sale.address` template.

Reply:
You're right! I have edited the template accordingly and tested the module's functionality and it works as expected.